CONTRIBUTING.md

```diff
@@ -176,6 +182,23 @@ You can also participate in the discussions at the following link: [discuss.daml
    If the dialog doesn't appear for you, you've probably already excercised one of these two choices.  
    To check your Firewall settings go to: `System Preferences` -> `Security & Privacy` -> `Firewall` -> `Firewall Options...` (checked on macOS Big Sur 11.5.2).  
   
+# IntelliJ Setup
+
+## Organize imports settings
+
+Go to `Menu -> Editor -> Code Style -> Scala` and then:
+1. Go `Imports` tab and in the `Import Layout` section make sure you have the folowing layout:
+    ```
+    base package imports
+    java
+    javax
+    _______ blank line _______
+    all other imports
+    _______ blank line _______
+    scala
+    
+    ```
+1. Set `Formatter` to `Scalafmt` and select `Configuration` file: `.scalafmt.conf`.
```

Comment:
The order of actions is inverted. Changing imports order must come after selecting a formatter